docs/deployment_guide/partner_editable/post_deployment.adoc

```diff
@@ -35,6 +35,34 @@ The deployment creates a security group that is used by Firewall Manager only as
 
 Firewall Manager creates a web ACL in each account that's managed by Firewall Manager.
 
+=== AWS Firewall Manager AWS Network Firewall policies
+
+* The Resource type for a Network Firewall policy is always VPC
+
+* Resources allows you to narrow the scope of your policy by specifying resource tags for inclusion. To use tagging, you need to first tag your resources. In Your VPCs, you use Resource tags with **key**: **__sumo-awsqs-nf-policy__** and **value**: **__network-firewall__**. For more information about tagging your resources, see https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/tag-editor.html[Working with Tag Editor]
+
+[#awsvpcresourcetags]
+.VPC Resource Tags
+[link=images/aws-vpc-networkfirewall-20240226110602.png]
+image::../docs/deployment_guide/images/aws-vpc-networkfirewall-20240226110602.png[AWS VPC Resource Tags^]
+
+=== AWS Firewall Manager​ AWS WAF policies
+
+For Policy tags, you can add any identifying tags that you want for the Firewall Manager AWS WAF policy. For more information about tags, see https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/tag-editor.html[Working with Tag Editor].
```

Comment:
Lets add below description

[Firewall Manager WAF Policies](https://docs.aws.amazon.com/waf/latest/developerguide/waf-policies.html) allow Firewall Manager to centrally mandate the application of AWS WAF Rule sets and Web ACLs to endpoints (ELBs, CloudFront Distributions, and API Gateway) in the environment.

remove `For Policy tags, you can add any identifying tags that you want for the Firewall Manager AWS WAF policy.` they need to add specifically our tags.